packages/workflows/__tests__/__system__/cli/List/activeWorkflows/cli.workflows.list.activeWorkflows.system.test.ts

```diff
@@ -0,0 +1,113 @@
+        system = testEnvironment.systemTestProperties.workflows.system;
+        owner = defaultSystem.zosmf.user;
+        wfName = `${getUniqueDatasetName(owner)}`;
+        definitionFile = `${defaultSystem.unix.testdir}/${getUniqueDatasetName(owner)}.xml`;
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+    });
+
+    afterAll(async () => {
+        await DeleteWorkflow.deleteWorkflow(REAL_SESSION, wfKey);
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+    describe("List all workflows", () => {
+        beforeAll(async () => {
+            // Upload files only for successful scenarios
+            await Upload.fileToUSSFile(REAL_SESSION, workflow, definitionFile, true);
+
+            // Create a workflow to list
+            await CreateWorkflow.createWorkflow(REAL_SESSION, wfName, definitionFile, system, owner);
+        });
+        afterAll(async () => {
+            let error;
+            let response;
+
+            const endpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_USS_FILES;
+            // deleting uploaded workflow file
+            try {
+                const wfEndpoint = endpoint + definitionFile;
+                response = await ZosmfRestClient.deleteExpectString(REAL_SESSION, wfEndpoint);
+            } catch (err) {
+                error = err;
+            }
+
+            response =  await ZosmfRestClient.getExpectJSON<IWorkflows>(REAL_SESSION, "/zosmf/workflow/rest/1.0/workflows?workflowName=" + wfName);
+            response.workflows.forEach(async (element: any) => {
+                if(element.workflowName===wfName){
+                    wfKey = element.workflowKey;
+                    try {
+                        await DeleteWorkflow.deleteWorkflow(REAL_SESSION, wfKey);
+                    } catch (err) {
+                        error = err;
+                    }
+                }
+            });
+        });
+        describe("Success Scenarios", () => {
+            it("Should return list of workflows in zOSMF.", async () => {
+                const response = runCliScript(__dirname + "/__scripts__/command/command_list_workflow.sh",
+                testEnvironment, [wfName]);
+                expect(response.stderr.toString()).toBe("");
+                expect(response.status).toBe(0);
+                expect(response.stdout.toString()).toContain(`${wfName}`);
+            });
+            it("Should return a message if search does not match any existing workflows", async () => {
+                const fakeName = `${wfName}${wfName}${wfName}`;
+                const response = await runCliScript(__dirname + "/__scripts__/command/command_list_workflow.sh",
+                testEnvironment, [fakeName]);
+                expect(response.status).toBe(0);
+                expect(response.stdout.toString()).toContain("No workflows match the requested querry");
+            });
+        });
+        describe("Display Help", () => {
```

Comment:
Help tests should be under `__integration__` instead of `__system__` so that they can automatically be run by the Jenkins machine